python/llm/src/ipex_llm/utils/benchmark_util_4_43.py

```diff
@@ -0,0 +1,4570 @@
+# coding=utf-8
```

Comment:
Use BIGDL license here, and add a reference to huggingface's utils.py, like 4_29.